osu.Game/Screens/Select/PlaySongSelect.cs

```diff
@@ -82,28 +88,48 @@ protected override bool OnStart()
         {
             if (player != null) return false;
 
-            // Ctrl+Enter should start map with autoplay enabled.
             if (GetContainingInputManager().CurrentState?.Keyboard.ControlPressed == true)
             {
-                var auto = Ruleset.Value.CreateInstance().GetAutoplayMod();
-                var autoType = auto?.GetType();
-
                 var mods = Mods.Value;
 
-                if (autoType == null)
+                Mod mod;
+                Type modType;
+
+                if (GetContainingInputManager().CurrentState?.Keyboard.ShiftPressed == true)
                 {
-                    notifications?.Post(new SimpleNotification
+                    // Ctrl + Shift + Enter should start map with cinema mod enabled.
+                    mod = Ruleset.Value.CreateInstance().GetAllMods().OfType<ModCinema>().FirstOrDefault();
+                    modType = mod?.GetType();
+
+                    if (modType == null)
                     {
-                        Text = "The current ruleset doesn't have an autoplay mod avalaible!"
-                    });
-                    return false;
+                        notifications?.Post(new SimpleNotification
+                        {
+                            Text = "The current ruleset doesn't have a cinema mod avalaible!"
+                        });
+                        return false;
+                    }
                 }
-
-                if (mods.All(m => m.GetType() != autoType))
+                else
                 {
-                    Mods.Value = mods.Append(auto).ToArray();
-                    removeAutoModOnResume = true;
+                    // Ctrl + Enter should start map with autoplay mod enabled.
+                    mod = Ruleset.Value.CreateInstance().GetAutoplayMod();
+                    modType = mod?.GetType();
+
+                    if (modType == null)
+                    {
+                        notifications?.Post(new SimpleNotification
+                        {
+                            Text = "The current ruleset doesn't have an autoplay mod avalaible!"
+                        });
+                        return false;
+                    }
                 }
+
+                if (mods.All(m => m.GetType() != modType))
+                    Mods.Value = mods.Append(mod).ToArray();
+
+                removeAutoModOnResume = true;
```

Comment:
This doesn't only remove the auto mod on resume any more. Variable rename is warranted here if this is to stay this way.